### PR TITLE
#365 remove prflight check

### PR DESCRIPTION
**Pull request by otselnik**
New `RETRY_ON_BLOCKED` variable defining count of retries on blocked accounts. In `environment.py` it is set to a minimum of 1.
Removed `RuntimeError` from `confirm_multiple_transactions`.
`collect_results` and `collect_result` now return response["result"]. And all corresponding changes by removing ['result']
Write sent by packs of 20.
CombinedContinue sent by 16.
Iterative execution performed By trying to send bucked. If got error of blocked or steps exceeded - retry until success and than go back to bucked send. If there were no errors or result retry bucked.
If got error and no succec transactions sent just return error. If were success trxs try cancel.

### CODE REVIEW

File: proxy/common_neon/transaction_sender.py [hunk 1]
```diff
@@ -349,108 +376,157 @@ def call_signed_iterative_combined(self):
...
-                result = self.sender.send_measured_transaction(trx, self.eth_trx, self.instruction_type)
-                return result
-            except SendTransactionError as err:
-                if check_if_program_exceeded_instructions(err.result):
-                    step_count = int(step_count * 90 / 100)
+            logs = []
+            try_one_step = False
+            found_errors = False
+
+            logger.debug(f"Send bucked combined: {self.instruction_type}")
```
Comment (ivandzen) [inline proxy/common_neon/transaction_sender.py]: Did you mean 'bucket'?

File: proxy/common_neon/solana_interactor.py [hunk 2]
```diff
@@ -131,13 +130,10 @@ def _getAccountData(self, account, expected_length, owner=None):
     def send_transaction(self, trx, eth_trx, reason=None):
         for _i in range(RETRY_ON_FAIL):
             reciept = self.send_transaction_unconfirmed(trx)
```
Comment (ivandzen) [inline proxy/common_neon/solana_interactor.py:132]: Why don't we catch exception here? send_transaction_unconfirmed still can throw error

Comment (otselnik) [reply proxy/common_neon/solana_interactor.py:132]: We only caught "could not confirm transaction" exception which is removed.

Comment (otselnik) [reply proxy/common_neon/solana_interactor.py:132]: If send_transaction_unconfirmed throws so be it.

**Review by ivandzen — APPROVED** [review]

**Pull request merged**